.github/workflows/CI.yml

```diff
@@ -263,3 +263,30 @@ jobs:
         with:
           command: rustdoc
           args: --features full,ffi -- --cfg docsrs --cfg hyper_unstable_ffi -D broken-intra-doc-links
+
+  check-external-types:
+    name: Check exposed types
+    needs: [style, test]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

Comment:
This action is [unmaintained](https://github.com/actions-rs/toolchain/issues/216). It seems like people are coalescing around [`dtolnay/rust-toolchain`](https://github.com/dtolnay/rust-toolchain) as an alternative solution.